package.json

```diff
@@ -112,10 +109,6 @@
           }
         ]
       }
-    },
-    "sql": {
-      "transitive_localized_views" : false,
```

Comment:
We need to check if this config fixes the issue on H2.

Reply:
Reintroducing this config did not help for H2.

Reply:
I think we should look at this again. I don't expect that this should be part of the default configuration.

Reply:
The default ("transitive_localized_views" : false) to be working. We don't need to re-introduce the config.